File: zjazd2/src/main/java/com/example/zjazd2/ui/quiz/Question.java
package com.example.zjazd2.ui.quiz;

public class Question {
    private String questionText;
    private String option1;
    private String option2;
    private String option3;
    private String option4;
    private int correctOption;

    public Question(String questionText, String option1, String option2, String option3, String option4, int correctOption) {
        this.questionText = questionText;
        this.option1 = option1;
        this.option2 = option2;
        this.option3 = option3;
        this.option4 = option4;
        this.correctOption = correctOption;
    }

    public String getQuestionText() {
        return questionText;
    }

    public String getOption1() {
        return option1;
    }

    public String getOption2() {
        return option2;
    }

    public String getOption3() {
        return option3;
    }

    public String getOption4() {
        return option4;
    }

    public int getCorrectOption() {
        return correctOption;
    }

    public String[] getOptions() {
        return new String[]{option1, option2, option3, option4};
    }
}

File: zjazd4/src/main/java/com/example/zjazd4/ui/quiz/Question.kt
package com.example.zjazd4.ui.quiz

class Question(s: String, s1: String, s2: String, s3: String, s4: String, i: Int) {
    private var questionText: String? = null
    private var option1: String? = null
    private var option2: String? = null
    private var option3: String? = null
    private var option4: String? = null
    private var correctOption = 0

    fun getQuestionText(): String? {
        return questionText
    }

    fun getCorrectOption(): Int {
        return correctOption
    }

    fun getOptions(): Array<String?> {
        return arrayOf(option1, option2, option3, option4)
    }
}